### DISCUSSION TITLE
tts_to_file speed is not changing despite gpu=True parameter

### DISCUSSION THREAD

**Discussion by mkordaci** [coqui-ai/TTS#2237] [General Q&A]
Hi everyone,

While I was trying the model "tts_models/en/vctk/vits" with gpu=False it takes nearly 13 seconds to complete tts_to_file method. Input is two sentences with 25 words in total.

Then I tried with gpu=True parameter after adding a nvidia T4 gpu to my cloud machine. I expected at least small improvement but it takes same amount of time.

Machine has 2vcpu and 8gb of memory with a T4.

Is that speed normal or do I have anything to add/change to improve speed?

Thanks

**Comment by p0p4k** [comment]
check if pytorch is cuda available returns true (google search for the exact syntax).
